src/bindings/c/include/openvino/c/ov_model.h

```diff
@@ -108,40 +111,47 @@ ov_model_reshape_input_by_name(const ov_model_t* model,
  * @ingroup model
  * @param model A pointer to the ov_model_t.
  * @param partialShape A PartialShape.
+ * @return Status code of the operation: OK(0) for success.
  */
 OPENVINO_C_API(ov_status_e)
-ov_model_reshape_one_input(const ov_model_t* model, const ov_partial_shape_t* partial_shape);
+ov_model_reshape_single_input(const ov_model_t* model, const ov_partial_shape_t* partial_shape);
 
 /**
  * @brief Do reshape in model with a list of <port id, partial shape>.
  * @ingroup model
  * @param model A pointer to the ov_model_t.
  * @param ports The port list.
```

Comment:
I think documentation should be updated to something like:
The array of port indexes

Just because at the current moment we have a mess in the documentation for what is node, port or port index :)

Reply:
ok, fix it.